### PR TITLE
mo5_flop: Metadata cleaning

### PR DESCRIPTION
**Pull request by ArcadeShadow**
Lower case on "Alt", "Hacked", "Format" words in the set description.
Removed empty lines.

### CODE REVIEW

**Review by cuavas — CHANGES_REQUESTED** [review]
Please leave the blank lines between the metadata and parts.  Removing them is unnecessary churn.

**Comment by ArcadeShadow** [comment]
> Please leave the blank lines between the metadata and parts. Removing them is unnecessary churn.

OK. Done it.

File: hash/mo5_flop.xml [hunk 1]
```diff
@@ -850,10 +850,11 @@ Thanks to DCMOTO (http://dcmoto.free.fr) for info!
 	</software>
 
 	<software name="mandrago">
-		<description>Mandragore (Hacked by Prehisto)</description>
+		<description>Mandragore (hacked)</description>
 		<year>1984</year>
 		<publisher>Infogrames</publisher>
-
+		<info name="hacked" value="Prehisto" />
+
```
Comment (cuavas) [inline hash/mo5_flop.xml:857]: Why does it show a diff for the blank line here?  Is there trailing whitespace?

Comment (einstein95) [reply hash/mo5_flop.xml:857]: Looks like github being weird, doesn't appear in https://github.com/mamedev/mame/pull/10125.patch

Comment (ArcadeShadow) [reply hash/mo5_flop.xml]: > Why does it show a diff for the blank line here? Is there trailing whitespace?

You're right. Blank spaces were left behind.
I removed them.
Now it's OK.

**Pull request merged**